.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2.1.4
         with:
-          node-version: '14.x'
+          node-version: '16.x'
```

Comment:
Can we go even higher?

Reply:
yeah node 16 will get deprecated from lambda runtime soon https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html. I did node 20 it worked.